docs/content/principles.mdx

```diff
@@ -3,36 +3,71 @@ name: Principles
 route: /principles
 ---
 
-# Principles
+## Design Principles
 
-## Obvious and transparent naming
+## Development Principles
```

Comment:
Happy to get input and ideas about this section with concrete guidelines on our component names, APIs, and conventions.